src/pocketmine/Player.php

```diff
@@ -2976,6 +2978,27 @@ public function handleDataPacket(DataPacket $packet){
 	}
 
 	/**
+	 * Transfers a player to another server.
+	 *
+	 * @param string $address The IP address or hostname of the destination server
+	 * @param int    $port    The destination port, defaults to 19132
+	 *
+	 * @return bool if transfer was successful.
+	 */
+	public function transfer(string $address, int $port = 19132) : bool{
+		$this->server->getPluginManager()->callEvent($ev = new PlayerTransferEvent($this, $address, $port));
+		if(!$ev->isCancelled()){
+			$pk = new TransferPacket();
+			$pk->address = $ev->getAddress();
+			$pk->port = $ev->getPort();
+			$this->dataPacket($pk);
+			$this->close($this->getDisplayName() . " transferred to another server", "transferred to another server", false); //TODO: add language strings
+			return true;
+		}
+		return false;
```

Comment:
Note: Consider returning `$ev` instead, in case the caller wants to know the real final destination.